### PR TITLE
refactor :  poddoor file and shutters file.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
1-added a folder for poddoor and shutters.
2-move those files inside their folders.
3-extracting almayer and two tile children of poddoor and give them their own files.

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
refactor: refactored poddoor file and shutters file.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by Huffie56** [comment]
as far as i know conflict was due to https://github.com/cmss13-devs/cmss13/pull/5892
so i implemented the fix into my change.

File: code/game/machinery/doors/poddoor/poddoor.dm [hunk 1]
```diff
@@ -0,0 +1,118 @@
+
+/obj/structure/machinery/door/poddoor
```
Comment (SabreML) [inline code/game/machinery/doors/poddoor/poddoor.dm:2]: Empty line here. (It's there in the original file too, but may as well remove it in this)
```suggestion
/obj/structure/machinery/door/poddoor
```

File: code/game/machinery/doors/poddoor/two_tile.dm [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	f2 = new/obj/structure/machinery/door/poddoor/filler_object (get_step(src,dir))
+	f1.density = density
+	f2.density = density
+	f1.set_opacity(opacity)
+	f2.set_opacity(opacity)
+
+/obj/structure/machinery/door/poddoor/two_tile/Destroy()
+	QDEL_NULL(f1)
+	QDEL_NULL(f2)
+	return ..()
```
Comment (SabreML) [inline code/game/machinery/doors/poddoor/two_tile.dm:91]: Could you move these to the top of the file? I'm not sure why they were all the way down here in the first place really.
(specifically `/obj/structure/machinery/door/poddoor/two_tile`, `/obj/structure/machinery/door/poddoor/two_tile/Initialize()`, and `/obj/structure/machinery/door/poddoor/two_tile/Destroy()`.)

**Review by SabreML — COMMENTED** [review]
Sorry for the delay! Just a couple of things.

**Comment by SabreML** [comment]
🤔

**Pull request merged**